.drone.yml

```diff
@@ -208,6 +214,9 @@ steps:
   - name: test-mssql
     pull: always
     image: golang:1.14
+    volumes:
```

Comment:
Doesn't this work without volume and with `tmpfs:` option

Reply:
@lafriks I'm not sure, I know _this_ works at least. 🤔 The difference AFAICT is that for the `tmpfs` option you must specify a target size (default: 64MB), whilst with this option it docker uses the available RAM (in my tests it was 1.95GB, because I've assigned 3GB to Docker Desktop). I wonder if reserving any amount of RAM for `tmpfs` would count against the memory available for the processes themselves. Also, I believe that `tmpfs` has a different lifespan.

Reply:
@lafriks It looks like the `--tmpfs` option as you proposed is the only way, since we need a `/tmp` with `exec` permissions (my PR as it is now makes it `noexec`).